src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -345,7 +348,10 @@
 
         return new NotificationHandler(
             (callbackStage, callback) -> callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                    && callback.getEvent() == Event.SUBMIT_APPEAL, notificationGenerators
+                    && Arrays.asList(
+                            Event.SUBMIT_APPEAL,
+                            Event.PAY_AND_SUBMIT_APPEAL)
```

Comment:
this covers happy path only, we need to have error handling in follow up story

Reply:
Noted